source/bdDetect.py

```diff
@@ -655,3 +666,8 @@ def addDeviceScanner(
 		scanForDevices.register(scanFunc)
 		if moveToStart:
 			scanForDevices.moveToEnd(scanFunc, last=False)
+
+
+class DeviceMathcer:

```

Comment:
Please explain the intention of this class. May be the name is too general:

```suggestion
class FallbackDevicesStore:
```